src/main/java/com/google/api/codegen/config/PackageMetadataConfig.java

```diff
@@ -59,6 +61,9 @@
 
   protected abstract Map<TargetLanguage, Map<String, VersionBound>> protoPackageDependencies();
 
+  @Nullable
+  protected abstract Map<TargetLanguage, Map<String, VersionBound>> protoPackageTestDependencies();
```

Comment:
Is it possible to make this empty map instead of null?

Guava also have Table, which is essentially 2D map, but looks like we should stick to map of map to look like protoPackageDependencies.